src/averageAges.js

```diff
@@ -15,11 +16,15 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
```

Comment:
Use named function expressions instead of function declarations. `const calculateMenAverageAge = () => {}`